### ISSUE TITLE
Spring Boot Prometheus AutoConfiguration is not working

### ISSUE THREAD

**Issue by RobWin** [resilience4j/resilience4j#161]
Adding only this two dependencies

```
        <dependency>
            <groupId>io.github.resilience4j</groupId>
            <artifactId>resilience4j-spring-boot</artifactId>
            <version>0.10.0</version>
        </dependency>
        <dependency>
            <groupId>io.github.resilience4j</groupId>
            <artifactId>resilience4j-metrics</artifactId>
            <version>0.10.0</version>
        </dependency>
```

results in the next message after starting:

```
java.lang.IllegalStateException: Could not evaluate condition on org.springframework.boot.actuate.autoconfigure.EndpointWebMvcAutoConfiguration$EndpointWebMvcConfiguration due to io/github/resilience4j/prometheus/RateLimiterExports not found. Make sure your own configuration does not rely on that class. This can also happen if you are @ComponentScanning a springframework package (e.g. if you put a @ComponentScan in the default package by mistake)
```

**Comment by RobWin** [comment]
@ciberado Could you please try `0.10.1-SNAPSHOT`.

**Comment by ciberado** [comment]
Works like a charm. Thanks!

**Comment by RobWin** [comment]
Thx, I release a new version tomorrow.

**Comment by RobWin** [comment]
I've released v0.10.1
